data_steward/retraction/retract_utils.py

```diff
@@ -134,6 +135,36 @@ def get_datasets_list(project_id, dataset_ids_str):
     return dataset_ids
 
 
+def is_deid_label_or_id(client, project_id, dataset_id):
+    """
```

Comment:
I think the function names are hard to follow here.  If we want to push people to rely on label over name conventions, the function definition is correct.  I would suggest naming `is_deid_label_or_id` to just `is_deid_dataset`.  The name `is_labeled_deid` is good.   And rename the current `is_deid_dataset` to something like `is_named_deid`.  Then `is_labeled_deid` and `is_named_deid` are helper functions to `is_deid_dataset`.  We could even make the helper functions protected and force code to use `is_deid_dataset` (which most code is already using).  Perhaps this is better as a subtask.

Reply:
Yes, I agree this should be a separate task. Adding the `is_deid_label_or_id` function already felt a bit scope-creepy but changing all usages of existing functions seems way out of scope.

Reply:
I'll create a ticket for this work once Jira is back up.